src/api/DummyApiResponder.php

```diff
@@ -413,6 +413,19 @@ protected function get_interface_response_editForumPost($args) {
     // End of Forum-related methods
     ////////////////////////////////////////////////////////////
 
+    protected function get_interface_response_loadDieSkillsData($args) {
+        return $this->load_json_data_from_file(
+            'loadDieSkillsData',
+            $args['threadId'] . '.json'
+        );
+    }
+
```

Comment:
Too much copy-paste.  There isn't going to be a 'threadId' arg, so this is going to give you a file with a useless name.  Try copying from an API that doesn't take args, e.g. `get_interface_response_loadOpenGames` is a good one to parrot.